lang/elixir/Portfile

```
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        elixir-lang elixir 1.15.6 v
+github.setup        elixir-lang elixir 1.15.7 v
 github.tarball_from archive
 epoch               1
```

Can you add a `revision` field?

```suggestion
revision            0
epoch               1
```

---

Sure. The current preferred format is to always include revision? At some point we wanted to skip it when it was 0.

---

I think I've recently seen that we want to have it explicit and present, even if set to zero.